cli/options/options.go

```diff
@@ -239,7 +241,11 @@ func HandleLoggingParams(debug bool, cfg config.ApplicationConfiguration) (*zap.
 	cc.DisableStacktrace = true
 	cc.EncoderConfig.EncodeDuration = zapcore.StringDurationEncoder
 	cc.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
-	cc.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	if term.IsTerminal(int(os.Stdout.Fd())) {
+		cc.EncoderConfig.EncodeTime = zapcore.TimeEncoderOfLayout("")
+	} else {
+		cc.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	}
```

Comment:
It should be vice-versa. If we use TTY (usual I do this as a developer), then we need to see the time.